docs/gestor-de-dependencias.md

```diff
@@ -0,0 +1,4 @@
+# **Gestor de dependencias**
+
+Para el gestor de dependencias he elegido Go Modules, está disponible para versiones de Go >= 1.16 y permite tener un control de las versiones de las dependencias, automatizar su instalación y limpiarlas cuando se dejan de usar de una forma muy fácil.   
```

Comment:
Creo que os lo he dicho varias veces. No me contéis lo que habéis elegido. Esto no es un TFG. 
Este objetivo trata de probar si tenéis madurez para tomar una serie de decisiones técnicas. Decir qué decisión habéis tomado sin contexto no prueba de ninguna forma vuestra madurez en ese sentido.